.travis.yml

```diff
@@ -7,8 +7,8 @@ services:
   - docker
 
 script:
-  - cd tools/docker/builder
-  - sudo ./image-build.sh
+  - cd tools/docker
+  - sudo ./image-build.sh -b ubuntu:18.04
```

Comment:
This is a reason to default to ubuntu:18.04 instead of defaulting to "native". I expect the native will not work in many cases, for various reasons. For example, it will not work on any non-Debian-derivative distro, because they don't have `apt-get`. It will also not work on any distro which has (slightly) different package names. 

However, enough of this docker stuff...